src/MoBi.Presentation/UICommand/NewModuleWithBuildingBlocksUICommand.cs

```diff
@@ -1,10 +1,11 @@
 using MoBi.Core.Domain.Model;
 using MoBi.Presentation.Tasks.Interaction;
+using OSPSuite.Presentation.MenuAndBars;
 using OSPSuite.Presentation.UICommands;
 
 namespace MoBi.Presentation.UICommand
 {
-   public class NewModuleWithBuildingBlocksUICommand : ObjectUICommand<MoBiProject>
+   public class NewModuleWithBuildingBlocksUICommand : IUICommand
```

Comment:
@rwmcintosh I am changing this, as I do not see the reason why we need the project as part of this command. Have tested and works fine.